v1/pipeline/steps/custom/harness-approval-step-info.yaml

```diff
@@ -4,9 +4,11 @@ allOf:
 - type: object
   required:
   - approvers
+  - message
   properties:
     callback_id:
       type: string
+      pattern: "^[a-zA-Z0-9_-]*(<\\+[^ ]+>)?[a-zA-Z0-9_.()-]*$"
```

Comment:
why do we need pattern?

Reply:
Ticket: https://harness.atlassian.net/browse/CDS-87088
We decided to only support alphanumeric + `-` + `_` in this field as this is an identifier fieldand is used in API to approve/reject approvals based on callback id.